pkg/schedule/coordinator.go

```diff
@@ -312,6 +312,38 @@ func (c *Coordinator) drivePushOperator() {
 	}
 }
 
+// driveEvictSlownNodeScheduler is used to create evciting slow node scheduler if using raft-kv2 engine.
+func (c *Coordinator) driveEvictSlownNodeScheduler() {
+	defer logutil.LogPanic()
+	defer c.wg.Done()
+
+	ticker := time.NewTicker(runSchedulerCheckInterval)
+	defer ticker.Stop()
+	for {
+		select {
+		case <-c.ctx.Done():
+			log.Info("drive evict trend scheduler has been stopped")
+			return
+		case <-ticker.C:
+			// If the cluster was set up with `raft-kv2` engine, this cluster should
+			// open `evict-slow-trend` scheduler as default.
+			if c.GetCluster().GetStoreConfig().IsRaftKV2() {
```

Comment:
How do make sure the config already is loaded or synced from tikv. I think if it is reliable, it's also reliable in `c.initSchedulers()`.

Reply:
Nope, we cannot do it, as the perception on whether this cluster is raft-kv2 or not is triggered by a asynchronous timer in `syncStoreConfig`.